modules/authglobus/lib/Auth/Source/Globus.php

```diff
@@ -127,26 +134,16 @@ protected function getFullName($name) {
     */
     protected function doCurlRequest(
```

Comment:
Not sure if it is possible, but could we reorder some of these parameters so that we dont have to have so many calls with null's in the code?